src/Illuminate/Auth/Passwords/PasswordBrokerManager.php

```diff
@@ -82,10 +82,8 @@ protected function resolve($name)
      */
     protected function createTokenRepository(array $config)
     {
-        $key = $this->app['config']['app.key'];
-
-        if (Str::startsWith($key, 'base64:')) {
-            $key = base64_decode(substr($key, 7));
+        if (Str::startsWith($key = $this->app['config']->get('app.key'), $prefix = 'base64:')) {
```

Comment:
This line is a mess, the lines before the change were much more clear to be

Reply:
Thanks @jmarcher ! I found the similar implementation multiple place. But Now I have revert this.